### ISSUE TITLE
How to get started?

### ISSUE THREAD

**Issue by paul1149** [jersou/mouse-actions#22]
I'm on MX Linux 21 (Debian 11). The .deb gui installed fine, but I can't get it to do anything.

I created an action using a "C" shape, with the command "kill". It does nothing, and as soon as I try the action a context menu opens on the program.

I want to do things like pass keystrokes, minimize/maximize windows, close programs, etc - basically what I use Easystroke for. But like the author, I find it crashes the session much too often, so I have to look for an alternative such as this.

Thanks.

**Comment by pallaswept** [comment]
If you're using X11, Try to use xdotool to send the keystroke to perform the function you require.

>  as soon as I try the action a context menu opens on the program. 

This sounds like you might have left one of the modifier keys enabled for your action, so the right-click to start the gesture is just a right-click. Make sure that none of these boxes are blue like this:

![image](https://github.com/jersou/mouse-actions/assets/132128962/5379882c-bb28-4b6e-aa52-ae4513e09639)

One of them is always enabled when you create a new binding:

![image](https://github.com/jersou/mouse-actions/assets/132128962/4dceb368-4936-4cd5-b5ea-94c452e29f9b)

I'm guessing that's still blue and that's why your gesture is not being recognised as a gesture but as a normal right-click.

**Comment by Dez1gner** [comment]
I'm having trouble using the program as well. I set it up like this:
![Снимок экрана от 2024-03-21 11-01-39](https://github.com/jersou/mouse-actions/assets/38717527/671e0522-7b1e-4922-aa4f-f5584d4bffc0)

I expect to see Nautilus launch when I press and hold the left mouse button and swipe this way. But nothing happens. What am I doing wrong?

**Comment by pallaswept** [comment]
> What am I doing wrong?

it's not entirely obvious at first so don't feel bad :) You're starting your command in the context of mouse-actions, so if you just run `nautilus` like the above screenshot, then you get nautilus launching in the non-interactive shell of mouse-actions - where you can't see it. Try doing `bash -c 'nautilus'` to spawn a new shell for nautilus to run in, and you'll see it appear.

**Comment by Dez1gner** [comment]
![Снимок экрана от 2024-03-21 14-39-22](https://github.com/jersou/mouse-actions/assets/38717527/fc9585a4-da36-4a7a-b052-2e125074feb6)
Did I do it right? But there is no Nautilus startup. I do the movement the same way I do it in Easystroke - left click and move the cursor.

**Comment by pallaswept** [comment]
> Did I do it right? But there is no Nautilus start. I do the movement the same way I do it in Easystroke - left click and move the cursor.

I can't test left-click, but that looks correct. 

For right-click, this works for me: 
![image](https://github.com/jersou/mouse-actions/assets/132128962/c0b105c3-8fb4-4ec1-b247-12d3524061b2)

Did you start the server (green button) after saving this (blue button)? It may be necessary, depending on your setup.

**Comment by Dez1gner** [comment]
Thanks for the replies. Yes, I press "Start". I started Kubuntu from a thumb drive and tried to start Dolphin in KDE and it didn't work either. I don't get it... Maybe I'm pressing wrong somehow? But how can I do it wrong :) I have been using EasyStroke for many years. Gestures are an integral part of my daily workflow. I have erased more than one tip of my Wacom by doing gestures. How can I not do the gesture correctly while doing so, no way.

Is there any way to find out what the cause is? Run MouseActions from the console and see what happens or doesn't happen?

![2024-03-22_09-30](https://github.com/jersou/mouse-actions/assets/38717527/7c6d32f7-c71e-4536-be0a-cb2f40d24908)

**Comment by Dez1gner** [comment]
I started the program and then clicked "Start". This is what the console showed:
https://pastebin.com/y7jcD6zC

**Comment by pallaswept** [comment]
> I started the program and then clicked "Start". This is what the console showed: https://pastebin.com/y7jcD6zC

Yep, follow those instructions. For a more verbose explanation, see documentation for installation here: https://github.com/jersou/mouse-actions/#requirement-
